minitwit/README.md

```diff
@@ -1,16 +1,15 @@
-# Minitwit server
+# MiniTwit
 
-Run the WebApplication class to open web server.
+Run the WebApplication class to open the web server locally.
 
-Run SQLLite class to initialize database.
+The application is currently deployed at http://143.198.249.10/.
 
 Uses
-* Spark as a web application framework
+* Spark as a web application framework:
   https://sparkjava.com/documentation
-* Apache velocity
+* Apache velocity for HTML templates:
   https://velocity.apache.org/engine/2.3/user-guide.html
-* SQLite-JDBC
-  https://mvnrepository.com/artifact/org.xerial/sqlite-jdbc
+* PostgreSQL as a remote database: https://mvnrepository.com/artifact/org.postgresql/postgresql
```

Comment:
Maybe a link to PostgreSQLs webpage/docs would be appropriate as well if we are adding maven link